src/scripts/autoloads/api_handler.gd

```diff
@@ -6,7 +6,7 @@ var _req_res: ResotoAPI.Request
 var _resoto_api: ResotoAPI = ResotoAPI.new()
 
 var adress: String		= "127.0.0.1" setget set_adress
-var port: int			= 8900 setget set_port
+var port: int			= 8980 setget set_port
```

Comment:
This port will only work on  localhost - is that correct?

Reply:
This is just the default port. It will automatically use the one in the url. I use this default only for the native version currently.